apps/backend/src/modules/ai-assistant/libs/helpers/initial-chat/generate-questions-answers-prompt.ts

```diff
@@ -0,0 +1,35 @@
+import { ZERO_INDEX } from "~/libs/constants/constants.js";
+import {
+	type OpenAiRequestMessage,
+	OpenAIRoleKey,
+} from "~/libs/modules/open-ai/open-ai.js";
+import { type OnboardingQuestionEntity } from "~/modules/onboarding/onboarding.js";
+
+import { OpenAiInitialPromptTemplate } from "./generate-init-prompt-template.enum.js";
+
+function generateQuestionsAnswersPrompt(
+	userQuestionsWithAnswers: OnboardingQuestionEntity[],
+): OpenAiRequestMessage {
+	const questionsWithAnswers = userQuestionsWithAnswers.map(
+		(questionEntity) => {
+			const { answers, label: question } = questionEntity.toObject();
+			const answer = answers[ZERO_INDEX]?.label;
```

Comment:
```suggestion
			const answer = answers[FIRST_ITEM_INDEX]?.label;
```
I believe we have this constant in shared

Reply:
Not fixed